api/api/error_handler.py

```diff
@@ -55,6 +55,43 @@ def _cleanup_detail_field(detail: str) -> str:
     return ' '.join(str(detail).replace("\n\n", ". ").replace("\n", "").split())
 
 
+async def handle_expect_header(request: ConnexionRequest, exc: ContentSizeExceeded = None) -> ConnexionResponse:
```

Comment:
For consistency reasons, since all other handler's function name end in `_handler`:
```suggestion
async def expect_header_error_handler(request: ConnexionRequest, exc: ContentSizeExceeded = None) -> ConnexionResponse:
```

Probably, this won't receive a `ContentSizeExceeded` either but a `WazuhError`.